servicetalk-grpc-protoc/src/test/proto/test_conflict_enum3.proto

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright © 2022 Apple Inc. and the ServiceTalk project authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Test file name conflicts with a top level enum -> generate code into the right location
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "io.servicetalk.grpc.protoc.test.conflict.enums";
+
+service TestConflictEnum3Service {
+  rpc DoNothing (TestConflictEnum3Req) returns (TestConflictEnum3Resp);
+}
+
+// todo(scott): make first char lowercase
+// https://github.com/protocolbuffers/protobuf/issues/9653
```

Comment:
once https://github.com/protocolbuffers/protobuf/issues/9653 is fixed, these tests can be updated to validate behavior.